Description: Github issue #142, the same bug that #100
             Disabling MultiSrc in Build. A paliative fix until
             to the nex release. Thanks hipersayanX - upstream.
             https://github.com/webcamoid/webcamoid/issues/142
Author: Herbert Parentes Fortes Neto <hpfn@debian.org>
Last-Update: 2018-11-02
Index: webcamoid/libAvKys/Plugins/Plugins.pro
===================================================================
--- webcamoid.orig/libAvKys/Plugins/Plugins.pro
+++ webcamoid/libAvKys/Plugins/Plugins.pro
@@ -31,7 +31,7 @@ SUBDIRS += \
     DesktopCapture \
     Multiplex \
     MultiSink \
-    MultiSrc \
+#    MultiSrc \
     Probe \
     VideoCapture \
     VirtualCamera
